lib/ansible/modules/windows/win_product_facts.ps1

```diff
@@ -57,9 +57,59 @@ if (-not $product_key) {
     }
 }
 
+try {
```

Comment:
Why do you have this in a try/catch? I'm able to retrieve this object even under a non-admin process so I don't think we need this. It may be best to get the CIM object then set the vars to NA if the property we are trying to get is null.

Reply:
Personally I prefer to have a value of $null (null) instead.

Reply:
I used NA as I have seen it in use elswhere. But I can change it to $null, thats no problem.

The try/catch was put in to protect against errors caused by some weird behaviour with antivirus blocking some of the powershell calls and classing them as malicious (BitDefender Hyper Detect).